tidy3d/components/source.py

```diff
@@ -466,6 +466,15 @@ class ReverseInterpolatedSource(Source):
         "placement at the specified location using linear interpolation.",
     )
 
+    confine_to_bounds: bool = pydantic.Field(
+        False,
+        title="Confine to Analytical Bounds",
+        description="If ``True``, any source amplitudes which, after discretization, fall beyond "
+        "the analytical bounds of the source are zeroed out, but only along directions where "
```

Comment:
How about changing the word "bounds" to "bounding box", as "bounds" can refer to multiple things? Also mention that the bounding box is inclusive. So perhaps "fall outside the bounding box of the source are zeroed out, but... The bounding box is inclusive."